idaes/models/properties/modular_properties/state_definitions/electrolyte_states.py

```diff
@@ -85,7 +85,7 @@ def _apparent_species_state(b):
     b.mole_frac_phase_comp_true = Var(
         b.params.true_phase_component_set,
         initialize=1 / len(b.params.true_species_set),
-        bounds=(1e-20, 1.001),
+        bounds=(0, 1.001),
```

Comment:
There is a good reason why these were set to a small number instead of 0 - if mole fraction goes to zero many of the equation in the properties can become singular thus it is necessary to set the bound away from zero. In the rare cases where someone is trying to track mole fractions with values of less than 1e-20 they should change this bound in the model after creation (at which point I have to ask if that is even meaningful).